backend/server/models.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+from peewee import *
+
+if os.environ["ENV"] == "test":
+    db = SqliteDatabase(":memory:")
+else:
+    db = SqliteDatabase("sqlite.db")
+
+
+class Thread(Model):
+    thread_id = CharField()
+    user_id = CharField()
+    title = CharField()
+    preset = CharField()
+
+    class Meta:
+        database = db
+
+
+class Message(Model):
+    message_id = CharField()
+    role = CharField()
+    content = CharField()
+    thread = ForeignKeyField(Thread, backref="messages")
+
```

Comment:
使用したpresetを管理してもいいかも？と思います

```suggestion
preset = CharField()
```